doc/source/release/1.20.0-notes.rst

```diff
@@ -0,0 +1,12 @@
+.. currentmodule:: numpy
+
+==========================
+NumPy 1.20.0 Release Notes
+==========================
+
+Improvements
+============
+
+Improve `array_repr`
+----------------------------------------------
+Now `array_repr` returns nicely formatted repr for arrays with objects with multi-line reprs.
```

Comment:
This should go in a newsfragment file by itself. I don't have the docs to hand, but another reviewer not on mobile may remember where they are.

Reply:
It goes to `doc/release/upcoming_changes`, There is a readme in that directory explaining it. (the header `--` should probably match the title in length, I think sphinx likes to complain about it otherwise).

Reply:
@panpiort8 could you move the snippet?